bugbug/bugzilla.py

```diff
@@ -96,6 +96,14 @@ def set_token(token):
 
 
 def get_ids(params):
+    """Retrieves a list of bug IDs from Bugzilla based on the given parameters.
+
+    Args:
+        params: A dictionary of parameters to pass to the Bugzilla API.
```

Comment:
We should not be describing the data types in the docstrings. If it is needed, we could add that as a type hint.